utils/docker/Dockerfile.el.9

```diff
@@ -0,0 +1,206 @@
+# All rights reserved.
+#
+# 'recipe' for Docker to build an image of EL 9 based
+# environment for building the DAOS project.
+#
+
+# Pull base image
+ARG BASE_DISTRO=almalinux:9
+FROM $BASE_DISTRO
+LABEL maintainer="daos@daos.groups.io"
+
+# Intermittent cache-bust.  Used to reduce load on the actual CACHEBUST later.
+ARG CB0
+
+# Use local repo server if present
+ARG REPO_FILE_URL
+RUN set -e;                                                      \
+    if [ -n "$REPO_FILE_URL" ]; then                             \
+        cd /etc/yum.repos.d/ &&                                  \
+        curl -k -f -o daos_ci-el9-artifactory.repo.tmp           \
+             "$REPO_FILE_URL"daos_ci-el9-artifactory.repo &&     \
+        for file in *.repo; do                                   \
+            true > $file;                                        \
+        done;                                                    \
+        mv daos_ci-el9-artifactory.repo{.tmp,};                  \
+    fi;                                                          \
+    dnf --disablerepo \*epel\* -y install epel-release           \
+                                          dnf-plugins-core;      \
+    if [ -n "$REPO_FILE_URL" ]; then                             \
+        PT_REPO=daos_ci-rocky8-crb-artifactory;                  \
+    else                                                         \
+        PT_REPO=crb;                                             \
+    fi;                                                          \
+    dnf -y config-manager --enable $PT_REPO;                     \
+    dnf -y clean all
+
+ARG JENKINS_URL
+ARG REPOS
+RUN for repo in $REPOS; do                                                \
+        branch="master";                                                  \
+        build_number="lastSuccessfulBuild";                               \
+        if [[ $repo = *@* ]]; then                                        \
+            branch="${repo#*@}";                                          \
+            repo="${repo%@*}";                                            \
+            if [[ $branch = *:* ]]; then                                  \
+                build_number="${branch#*:}";                              \
+                branch="${branch%:*}";                                    \
+            fi;                                                           \
+        fi;                                                               \
+        echo -e "[$repo:$branch:$build_number]\n\
+name=$repo:$branch:$build_number\n\
+baseurl=${JENKINS_URL}job/daos-stack/job/$repo/job/$branch/$build_number/artifact/artifacts/el9/\n\
+enabled=1\n\
+gpgcheck=False\n" >> /etc/yum.repos.d/$repo:$branch:$build_number.repo;   \
+        cat /etc/yum.repos.d/$repo:$branch:$build_number.repo; \
+        dnf -y repolist; \
+        dnf --disablerepo=\* --enablerepo=$repo:$branch:$build_number makecache; \
+    done
```

Comment:
Same applies this this chunk, unless you're running under Jenkins this won't activate and I suspect there's work to configure the servers before this can be used.

Reply:
Again, this is very tried and tested code that is in every Dockerfile.  In fact removing this will remove commit pragma support for EL9 which will lead to time wasted and queries about why commit pragmas are not working which will lead to even more wasted time to figure out why and more wasted time to resurrect it.  Again, I strongly disagree with this suggestion.

Reply:
I copied it from the EL8 file and would suggest any refactoring to be done in a different PR not managed by me :)